src/service/vsync-impl.js

```diff
@@ -309,6 +334,14 @@ export class Vsync {
         tasks[i].mutate(states[i]);
       }
     }
+    // Swap last arrays into double buffer.
+    this.nextTasks_ = tasks;
```

Comment:
Ha, I totally read this backwards (as in you're truncating the old `nextTasks_`, not the new). Never mind.